crates/scheduler/src/call/usb/serial.rs

```diff
@@ -95,7 +95,7 @@ fn unregister<B: Board>(mut call: SchedulerCall<B, api::unregister::Sig>) {
 #[cfg(feature = "board-api-usb-serial")]
 fn flush<B: Board>(call: SchedulerCall<B, api::flush::Sig>) {
     let api::flush::Params {} = call.read();
-    let result = try { board::usb::Serial::<B>::flush()? };
+    let result = try bikeshed _ { board::usb::Serial::<B>::flush()? };
```

Comment:
Curiosity on my part, thinking about possible designs: 

Can you say more about these single-`?`-at-the-end try blocks?  Are these doing error conversion between types?  Are they here for consistency?  Or...